.gitignore

```diff
@@ -1,2 +1,4 @@
 *.DS_Store
 *.o
+*.out
```

Comment:
good one. I would love to merge this but for the statement, I'll need a use case where it can be useless!